src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -76,193 +143,422 @@ abbr {
 pre, code {
     font-size: 0.9em;
 }
-.clear {
-    clear: both;
-    height: 0;
-    font-size: 0;
-    line-height: 0;
+
+.help {
+    color: #999;
+    font-size: 14px;
+}
+p.help {
+    margin-bottom: .5em;
+}
+.empty {
+    border: 4px dashed #E0E0E0;
+    color: #999;
+    margin: 1em 0;
+    padding: .5em 2em;
+    text-align: left;
+}
+
+.muted {
+    color: #999;
 }
-.clear-fix:after
-{
-    content: "\0020";
+.nowrap {
+    white-space: pre;
+}
+.newline {
     display: block;
-    height: 0;
-    clear: both;
-    visibility: hidden;
 }
-* html .clear-fix
-{
-    height: 1%;
+.break-long-words {
+    -ms-word-break: break-all;
+     word-break: break-all;
+     word-break: break-word;
+-webkit-hyphens: auto;
+   -moz-hyphens: auto;
+        hyphens: auto;
+}
+.small-text {
+    font-size: 12px;
+}
+.bold {
+    font-weight: bold;
+}
+.text-right {
+    text-align: right;
 }
-.clear-fix
-{
+
+.metric {
+    background: #FFF;
+    border: 1px solid #E0E0E0;
+    font-size: 16px;
+    min-width: 100px;
+    min-height: 70px;
+}
+.metric .value {
     display: block;
+    font-size: 28px;
+    padding: 5px 15px;
+    text-align: center;
 }
-#content {
-    padding: 0 50px;
-    margin: 0 auto 20px;
-    font-family: Arial, Helvetica, sans-serif;
-    min-width: 970px;
+.metric .value svg {
+    margin: 5px 0 -5px;
+}
+.metric .unit {
+    color: #999;
+    font-size: 18px;
+    margin-left: -4px;
+}
+.metric .label {
+    background: #E0E0E0;
+    display: block;
+    font-size: 12px;
+    font-weight: bold;
+    padding: 5px;
+    text-align: center;
+}
+.metrics {
+    margin: 1em 0;
+    overflow: auto;
+}
+.metrics .metric {
+    float: left;
+    margin-right: 1em;
+}
+
+.metrics-horizontal .metric {
+    min-height: auto;
+    min-width: auto;
+}
+.metrics-horizontal .metric .value,
+.metrics-horizontal .metric .label {
+    display: inline;
+    padding: 2px 6px;
+}
+.metrics-horizontal .metric .label {
+    display: inline-block;
+    padding: 6px;
+}
+.metrics-horizontal .metric .value {
+    font-size: 16px;
+}
+.metrics-horizontal .metric .value svg {
+    max-height: 14px;
+    line-height: 10px;
+    margin: 0;
+    padding-left: 4px;
+    vertical-align: middle;
+}
+
+table, .metric {
+    box-shadow: 0px 0px 1px rgba(128, 128, 128, .2);
+}
+
+.container {
+    min-width: 960px;
+    max-width: 1200px;
```

Comment:
In your screenshots it renders perfectly, but why limit the width ?

Because on a big screen this look like this:
![capture d ecran 2015-08-12 a 17 21 58](https://cloud.githubusercontent.com/assets/334432/9227977/8ca1de82-4116-11e5-8e29-3523f5d836d9.png)